crates/red_knot_python_semantic/resources/mdtest/with/with.md

```diff
@@ -115,10 +115,10 @@ class NotAContextManager: ...
 
 context_expr = Manager1() if coinflip() else NotAContextManager()
 
-# error: [invalid-context-manager] "Object of type Manager1 | NotAContextManager cannot be used with `with` because the method `__enter__` of type Literal[__enter__] | Unbound is not callable"
-# error: [invalid-context-manager] "Object of type Manager1 | NotAContextManager cannot be used with `with` because the method `__exit__` of type Literal[__exit__] | Unbound is not callable"
+# error: [invalid-context-manager] "Object of type `Manager1 | NotAContextManager` cannot be used with `with` because the method `__enter__` is potentially unbound"
```

Comment:
Nit: Most other diagnostics use the wording `possibly` over `potentially`. Both sound okay to me but I'm leaning towards the more consistent wording unless there's a specific reason you chose potentially

Reply:
Will change it to "possibly" everywhere